lib/fastlane/actions/create_keychain.rb

```diff
@@ -28,6 +28,15 @@ def self.run(params)
         command << " ~/Library/Keychains/#{escaped_name}"
 
         commands << Fastlane::Actions.sh(command, log: false)
+
+        if params[:add_to_search_list]
+          keychains = []
+          keychains += Fastlane::Actions.sh("security list-keychains -d user", log: false).shellsplit unless Helper.test?
+          keychains << File.expand_path(params[:name], "~/Library/Keychains")
+          keychains.uniq!
```

Comment:
This is a relic of the time before I added `-d user`. The result of the command would automatically include the system search list (i.e. `/Library/Keychains/System.keychain`), which I would then add to the user search list. Next time `/Library/Keychains/System.keychain` would appear twice in the search list, then thrice, and the list would just grow longer and longer each time.

But now, we can remove that line.